workflow/target.go

```diff
@@ -28,6 +28,7 @@ type Publishing struct {
 type CodeSamples struct {
 	Output   string          `yaml:"output"`
 	Registry *SourceRegistry `yaml:"registry,omitempty"`
+	Style    *string         `yaml:"style,omitempty"` // Oneof "standard", "readme" (default: standard) (see codesamples.go)
```

Comment:
did we need one for redocly? I forgot

Reply:
redocly uses the standard one https://redocly.com/docs/api-reference-docs/specification-extensions/x-code-samples